TESTING.md

```diff
@@ -24,58 +24,67 @@ Things to look at:
  
  *Blue:*
  - Selected/highlighted content
- - variable controls - progress bars, scales, level bars
+ - variable/ scaling controls - progress bars, scales, level bars
  - General Content-relevant controls
  
  *Orange:*
  - Binary on/off actions
  - radio buttons
- - active buttons
- - toggle switches
  - checkboxes
  - switches
  - Pathbars
+ - Focused entries
  - General context-display controls
   
  *Green:*
  - Save buttons once they can be clicked
  - Affirmative/progressive action buttons/controls
  
  *Header and title bars:* 
- - brown in normal theme
- - dark theme - darker brown
- - light theme - grey
+ - brown in normal theme (#574F4A)
+ - dark theme - dark grey (#363636)
+ - light theme - light grey (Light theme is currently missing)
```

Comment:
If it is currently missing, why don't we leave it out until such a theme is ready and then make a pr with those additions?

Reply:
Updated